mitiq/benchmarks/volume_circuits.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (C) 2022 Unitary Fund
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Functions for creating circuits of the form used in quantum
+volume experiments as defined in https://arxiv.org/abs/1811.12926.
+
+Useful overview of quantum volume experiments:
+https://pennylane.ai/qml/demos/quantum_volume.html
+
+Cirq implementation of quantum volume circuits:
+cirq-core/cirq/contrib/quantum_volume/quantum_volume.py
```

Comment:
Actually reusing part of this + some basic functioning would make a great Example for the example section in the Documentation, for another follow-up PR.